src/bijectors/ordered.jl

```diff
@@ -88,3 +78,72 @@ end
 
 logabsdetjac(b::OrderedBijector, x::AbstractVector) = sum(@view(x[2:end]))
 logabsdetjac(b::OrderedBijector, x::AbstractMatrix) = vec(sum(@view(x[2:end, :]); dims=1))
+
+# Need a custom distribution type to handle this properly.
+struct OrderedDistribution{B,D<:ContinuousMultivariateDistribution} <:
```

Comment:
Maybe the types should be ordered the same as the fields:
```suggestion
struct OrderedDistribution{D<:ContinuousMultivariateDistribution,B} <:
```